src/Perl6/Pod.nqp

```diff
@@ -116,32 +111,280 @@ class Perl6::Pod {
         ).compile_time_value
     }
 
+    sub decint($s) {
+        # code copied from Actions.nqp and locally modified
+        my int $base := 10;
+        my int $chars := nqp::chars($s);
+        $chars > $max-dec-intchars
+          ?? string_to_bigint($s, $base, $chars)
```

Comment:
Why not use a bigint *always*? Will make the code simpler.

Reply:
Can probably do so...will try it.

Reply:
So done—ready for another review.